### DISCUSSION TITLE
為自己製造快樂

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1185] [General]
有些人天生個性比較樂觀，而基因決定我們一部份的快樂。但其實快樂不只有基因決定而已，我們也可以自己製造快樂 :
1. 感恩擁有的每件事
2. 期望值設定在可接受範圍
3. 放下我們所失去的
有時快樂需要花時間尋找才知道自己怎樣才會感到開心~~
(https://sonjalyubomirsky.com/)

**Comment by GuanYann** [comment]
對我來說睡覺是一件很快樂的事情，雖然它不能解決任何問題，但能讓我暫時忘卻煩惱，並以更好的精神面對挑戰

**Comment by Lipunpun** [comment]
認同作者的觀點，對我來說，有錢花就是一件很快樂的事情，最好是不用出力的那種，給別人養也可以，有人缺乾兒子的嗎? 我最近要面試幾個乾爹、乾媽，可以藝一起來報名喔。

**Comment by zilinng** [comment]
除了上面的方法以外，許多研究都表明運動能夠促使大腦多巴胺的分泌，有助於產生愉悅感、舒緩壓力，讓我們變得快樂～所以如果心情不好的時候，不妨嘗試去運動，即便只是散散步，多少都有提振心情的效果喔！

**Comment by curtis1017** [comment]
如果人缺乏了快樂，那他的人生不在這麼地有意義，對我而言，儘管平時功課壓力很大，但我還是會跟同學忙裡偷閒，我們會不定時的吃宵夜跟夜唱，然後騎著摩托車到處跑。

**Comment by Elaine2899** [comment]
十分認同樓上說的嘗試運動來變得更快樂!!
這學期因為各種因素，無法到系隊運動打打球，同時也不知道為什麼總覺得沒有心情出去散步。心情郁悶、胸口有股不舒服的感覺揮之不去，更難過的是，我的腸胃經常因為我的心情起伏而變化。最近的emo也被腸胃感受到了，現在的我真的是"大腹便便" ，腸胃不適又讓我心情更差導致負面循環。
而上學期每周都有至少運動兩天，偶爾也會撥時間去操場跑跑，那時候雖然也有考試、工讀和其他壓力，但那些壓力和負面情緒總會在一個安全閥值之下。所以我覺得，運動真的使我變得快樂，而且讓我對自己更滿意、更有自信，也對其他事物更想向上努力。
段考周過後我一定要找回上學期的好習慣，也找回我的快樂!!
祝大家也找到適合自己的運動方式及頻率!!

**Comment by KevinCheng42** [comment]
我贊同作者的說法，感恩擁有每件事，知足，能讓我們感到快樂及幸福；另外，找到自己的興趣也可以增添快樂，當我們做自己熱愛的事情時，如打球或唱歌，抑或是參加社團等，我們會感到快樂並且對此富有熱情，也因此常常不自覺投入了許多時間～

**Comment by YoYooHuang** [comment]
快樂其實很簡單，快樂是源自於自身內心的感受。同樣的事情，不同的人會有不同的感受，有人也許感到憤怒、難過，但可能也有人因此感到快樂。最簡單的快樂就是時常保持微笑，用正能量面對每件事情，即使在別人眼中看起來微不足道的事情，也能讓自己感到快樂!!

**Comment by sjh501** [comment]
在壓力大或面對問題時，我也會空出一些找讓自己快樂的事情做。對我來說，雖然花一些時間讓自己快樂在表面來看並沒有實質上的解決問題，但放鬆心情、轉換不一樣的想法，使再次面對問題時更有效率，這也是間接的在解決問題。

**Comment by Evening218** [comment]
我平常在面對不想處理的事情時,都會先給自己訂好一個獎勵制度,比如做完一部份報告就可以打一場遊戲或看一章小說,雖然聽起來沒什麼意義甚至會導致分心,但對我來說,這能幫助我保持好的情緒狀態完成工作,甚至能提高效率或降低我擺爛的機率。"為自己製造快樂"是個很棒的說法。

**Comment by tower20021202** [comment]
快樂其實很簡單，有時候在路上巧遇朋友聊個天，又或是晚間繞校園跑步運動，都可以有效地讓我們自身感到快樂

**Comment by ziihui** [comment]
當我們感謝自己所擁有的一切，我們可以更容易地看到自己的幸福和滿足感。感恩可以提醒我們，我們身邊有很多值得感激的東西！

而我們也能透過感恩、放下失去的東西就能花更多心力尋找快樂，我們可以使自己獲得更多滿足。

**Comment by Jack-1D** [comment]
我認為保持快樂相當重要。在這個科技進步的時代，我們活在太容易獲取滿足的世界裡，如看個簡短的短影片，或是打一場手機遊戲都能獲得快樂。然而，這種太容易獲得的快樂並不會持久。相反的，這會讓我們陷入更大的不快樂反彈，我們也很容易因此變得難以快樂。
[https://www.youtube.com/watch?v=fOHdnxoscws&ab_channel=shuaisoserious](https://www.youtube.com/watch?v=fOHdnxoscws&ab_channel=shuaisoserious)

**Comment by llh26** [comment]
其實升大二以後，我很少真正感受到快樂，和大一相比之下真的少了很多，可能是因為對未來的不確定性變高，造成我雖然過得很忙碌，但總覺得自己活得毫無目標，加上自我懷疑的情緒，導致我常常看起來悶悶不樂，為自己製造快樂對我來說真的是個很重要的課題，希望我可以早日找到讓自己快樂的因素！

**Comment by PaulSu0905** [comment]
我覺得降低期望值是一個很好的方法，因為我就是一個很容易想太多的人，各種事情好的壞的方面我都先想過了，等到真的體驗時就覺得跟自己想的差不多而失去新鮮感，當試著先不思考太多，先實際體會，就能感受到更多感情，不管是快樂還是傷心都更加強烈。

**Comment by larry40305** [comment]
誰說用錢買不到快樂，如果有，那一定是不夠多。

**Comment by aa103967622** [comment]
我覺得人在努力，總是需要一些能夠讓自己感到快樂、覺得努力是件很值得去做的事情，這樣的獎勵機制。有了完成某件事後，我們給自己的獎勵，我們將會更有動力去向前邁進！

**Comment by bestbanana295** [comment]
我覺得一個人如果想要變跨了的話。必須要發自內心去做覺得自己會快樂的事情，因為只要有意思的假裝便會露出破綻，那麼就不能好好享受快了的時光了，希望每個人都可以找到自己的快樂。兒且可以 快樂的去過生秮。

**Comment by Aaron0810** [comment]
快樂其實很簡單，快樂是源自於自身內心的感受。同樣的事情，不同的人會有不同的感受，有人也許感到憤怒、難過，但可能也有人因此感到快樂。最簡單的快樂就是時常保持微笑，用正能量面對每件事情，即使在別人眼中看起來微不足道的事情，也能讓自己感到快樂!!
並且我也贊同作者的說法，感恩擁有每件事，知足，能讓我們感到快樂及幸福；另外，找到自己的興趣也可以增添快樂，當我們做自己熱愛的事情時，如打球或唱歌，抑或是參加社團等，我們會感到快樂並且對此富有熱情，也因此常常不自覺投入了許多時間～

**Comment by Miinteaa** [comment]
對於我來說，拋開煩惱是達成快樂最直接的方法。我曾在《被討厭的勇氣》中讀到，人們的一切煩惱跟不快樂都是因為人際關係而產生的，只要我們學習好「課題分離」，只控制自己的選擇，不去插手管別人的事，那麼煩惱跟在意的事情就會少很多，而快樂就會變多了！

**Comment by zhiron544** [comment]
這篇文章讓我思考了自己的快樂來源，我認為快樂是一種主觀的感受，每個人都有不同的定義和方法。我同意作者的觀點，我們可以通過積極的心態和行為來提升自己的幸福感，也可以從別人身上學習和感染。我想知道作者是如何發現自己的快樂方式的，是否有遇到困難和挑戰。

**Comment by Alex11392** [comment]
這三個方法就是說起來容易，要聽懂也不難，但要真的透過這樣感受到快樂就很不容易了

**Comment by ThanatosJun** [comment]
有一本書叫《運動改造大腦》，這是我在體育課當中接觸到的。運動使人大腦分泌激素，讓人愉悅，也讓人記憶力加深，可見適量運動的確能讓人感到心情上的快樂。然而快樂的提升值因人而異，就像打 RPG 遊戲一樣，不同的怪物適應不同的攻擊，能讓自身愉悅的事物只有自己才感受的到。

**Comment by jim1743** [comment]
在壓力大或是課程變煩的時候，我的解放壓力來源就是打遊戲，藉由打遊戲，抒發內心的壓力時就讓我很開心，另外其他會讓我快樂的事，不外乎就是看有趣的影片，藉由畫面帶來的娛樂效果，讓我暫時忘卻課業的煩惱。

**Comment by tzurou** [comment]
壓力很大的時候我會想要逃離現實，會一直看影片，拖延症也會變得很嚴重，但這樣的確會讓我感受到短暫的快樂，或是和朋友一起大笑也能紓解一直以來堆積的壓力，只要壓力越大我就會越用力地玩，通常這樣玩完以後我做事也會變得比較有效率。

**Comment by 110101501** [comment]
我認為快樂一直都在我們身邊，只是在於我們能不能感受，或是發現。
不見得一定要很明確物質上的快樂，看到美麗的風景、吃到好吃的食物，生活中的小事，都能成為我們的快樂來源。

**Comment by wanka2030** [comment]
快樂可以通過多種方式獲得。首先，關注積極的事物和感恩，記錄每天的美好瞬間。其次，保持健康的生活方式，包括均衡飲食、規律運動和充足睡眠。與親友保持良好的關係，參加社交活動，分享快樂和支持。找到自己熱愛的興趣或愛好，投入其中。學會管理壓力，採用冥想或深呼吸等放鬆技巧。最後，幫助他人可以帶來內心的滿足和快樂。通過這些方法，你可以逐漸提升幸福感。之前上課記錄下來的筆記，反正快樂無所不在，耐心尋找。

**Comment by YOGA012** [comment]
跑步對我來說不僅是一種運動，更是一種自我製造快樂的方式。每一次踏出步伐，我都能感受到身體的活力與自由，這種感覺讓我忘卻了一切煩惱與壓力。在跑步的過程中，我能夠專注於呼吸與步伐，感受著風吹過臉頰的清涼，享受著身體被活力充滿的感覺。跑步讓我更加感恩擁有健康的身體，並期望自己能持續享受這份快樂，而且跑步也讓我認識了一群有趣的同好，有他們的陪伴跑步變成了更加快樂的事。無論是在路上或賽道上，跑步都讓我找到了屬於自己的快樂。

**Comment by Mitochondrionjessica** [comment]
對我來說去認真經營生活，活在當下就是快樂的~

**Comment by YuanWang08** [comment]
快樂因人而異，不同人對於快樂的定義與方式不一樣，對我而言，只要生活還有目標，每天能感覺到自己比昨天更進步，我就感到很快樂了，不需要什麼過多的物質需求或認同需求，隨著年紀增長，快樂這個詞也越來越不一樣了。

**Comment by jylin34** [comment]
對我來說，讓自己快樂的方式就是專注在自己身上，努力朝目標一天一天邁進，不要想太多。 心頭有雜念的時候，去跑跑步 運動產生一點多巴胺之類的也會比較好。

**Comment by alexender0104** [comment]
我認為想要製造快樂，可以嘗試新的活動，如繪畫、音樂、園藝或運動，發現讓自己感到快樂和充實的興趣。或是與與親朋好友保持聯繫，共享生活中的點滴，這有助於提升情感支持和幸福感。

**Comment by wanka2030** [comment]
最近真的好需要快樂🎉被期末壓著打或者根本是完全壓制住，有好多事情沒有完成但還是會拿時間讓自己放鬆。禮拜天時也去聽了黃仁勳的演講，想到未來AI世界的驚奇就想要繼續活下去了，必須要努力讓自己努力完成每一件事情🔥

**Comment by hbh10601** [comment]
身為ESFJ的我真的是內耗冠軍，被身邊朋友不知道念過多少次，點點點

**Comment by yayun601506** [comment]
我有時候在學校待得很悶，就會一個人騎車去咖啡廳，我很享受一個人的悠閒時光，吃吃甜點心情會好一點

**Comment by 1b5b** [comment]
我覺得對i人來說，獨處也是一種快樂的來源

**Comment by tshsu04224** [comment]
我不算一個本來就十分樂觀的人，常常一件事想一想，想越久就想到越多悲觀的面向，但我的好處在於忘事情忘的很快，我可能今天想一件事想到心情很不好，但隔天睡醒就忘記昨天的悲觀，重新變成那個開朗的我了，健忘以這部分來說也算是一件還不錯的事吧

**Comment by lilishh** [comment]
身為一個算蠻樂觀的人，有時候心情低沉，其實也會一直糾結在這件事上，但每當遇到這件事時，我覺得獨處或是稍微睡個覺、看個自己喜歡的影片，都可以蠻成功的轉換心情。雖然常常被人說自己有點太樂觀，但我想這也不是壞事吧。

**Comment by 710388** [comment]
我平常都很快樂，感覺是在努力為別人製造快樂哈哈，希望別人也會為我製造快樂

**Comment by sandyho0307** [comment]
我覺得如果我們的期望值過高，達不到目標時就會感到失落和不快樂。相反，設定可接受的期望值，可以讓我們更容易實現目標，從而獲得滿足感和快樂。

**Comment by sunset513** [comment]
我也認同樓上的看法，期望值太高的話可能反而變成失落感太大。我覺得這種失落感甚至會讓我感到痛苦，像是人際互動時會期待對方的反應很積極，結果符合預期會很開心；但結果不如預期時就一定會相當失落。

另外我自已覺得我們可以多增加幾種自己的快樂來源，像是人際之外，我也會透過運動、鋼琴等興趣來讓自己收穫來樂。這種快樂的來源更穩定，也可以讓整體的快樂來源便更多，避免單一的來源影響太大。

**Comment by Shang621** [comment]
為自己製造快樂，是一種積極的生活態度。這並不是自私，而是懂得珍惜和照顧自己的需求。現代生活壓力繁重，學會在繁忙中找到快樂的來源至關重要。這可以是簡單的愛好，如閱讀、運動，或是與親友共度時光。快樂是內心的感受，不應依賴外在的條件或他人的認可。主動追求快樂，能增強自信心和生活滿足感，使人更有動力面對挑戰。保持積極心態，尋找生活中的小確幸，讓自己更有活力和幸福感。

**Comment by buoioiry** [comment]
我是一個蠻容易自娛的人，很多小事情都可以讓我開心很久，主要是如果一直不開心的話會心悸，感覺很傷身體，珍惜身邊的小確幸就更有動力做事

**Comment by jeff110303549** [comment]
我覺得製造快樂這件事情非常因人而異，並不是每一個人都有動力去正向的面對自己的生活，這和家庭背景等成長經歷有很大的相關性，不過我很認同版主說的感恩擁有的每一件事，不一定只是因為感恩而讓自己身心愉悅，而更多的事徹底的去感受、分析身邊所有的一切，或許有太多值得我們去厭惡、去難過的事，但卻一定也有不少身邊的人事物沒有任何利弊得失的為我們加油，每每看到或想到類似的事情就能讓我更有正向思考的動力

**Comment by cassy0201** [comment]
我覺得減少自己對某些事情的期望或慾望也是讓自己更快樂的方式,因為有了期望就會有失望，減少慾望後相對也更容易得到滿足

**Comment by 155Julia** [comment]
以我個人的觀點來看，我認為快樂是一種選擇和習慣。就像我自己一樣，作為一個ENFP，我傾向於以積極樂觀的態度面對生活中的挑戰，這讓我能夠更輕鬆地處理困難並快速恢復。我發現，當我感恩身邊的一切，設定合理的期望並學會釋放過去的失去時，我的快樂感更加持久。對我而言，快樂不僅是一種情緒，更是一種心態，一種選擇去積極面對生活，從中尋找喜悅和滿足。

**Comment by YeeDaniel** [comment]
快樂嗎，我覺得其實做自己喜歡做的事就很快樂欸，像我自己就很喜歡運動，這學期還去參加了鐵人三項和路跑，尤其在鐵人三項有得名，讓我提升了運動的成就感

**Comment by yubo0oo** [comment]
懷抱一顆感恩的心，珍惜擁有的一切得失心不要太重，天天都可以開心

**Comment by poiu6fi** [comment]
本身不算是一個很樂觀的人，會一直糾結在某件事上面，把自己搞得其他事都做不下去，休息一下轉移注意力我覺得是個不錯的方法

**Comment by yjpxi** [comment]
我覺得快樂並不會自動出現，我們需要主動去尋找。可以讓自己探索新的愛好、學習新的技能或是花時間與所愛的人在一起，透過這些方式，我們可以發現什麼讓自己真正感到快樂。

**Comment by peresonacifer** [comment]
身為學生，我認為我要獲取快樂還是挺容易的，最簡單直接的方式就是交朋友，尤其是找有共同興趣或共同話題的人，像是可以和一群喜歡打遊戲及講話口無遮攔的朋友混在一塊，他們能帶給你一天的快樂，但如果你想自己一個人清淨，那你可以選擇看看小說或動漫，然而，這種快樂不能長久，如果想要快樂能陪伴自己久一點，那發展自己的興趣是一大方法，畢竟他能給你動力和成就感，投入興趣的感覺難用言語形容但敢說一定快樂的。

**Comment by MingWei174** [comment]
我覺得每天和朋友聊聊天，珍惜跟家人互動的每個瞬間，就帶給我很大的喜悅，另外多從事自己感興趣的活動也可以為我帶來滿滿的舒適感以及趣味，我自己在遭遇生活中的不如意時，則是藉由玩遊戲來把一天累積的煩惱拋在腦後，暫時放輕鬆不要亂了自己的思考邏輯，玩完遊戲放鬆身心後，就可以更有效率地處理困難，為自己營造一個無憂無慮且充滿樂趣的生活。
[把握當下，及時行樂]，這句話對我來說是活出快樂人生的指標!

**Comment by Nana1218** [comment]
我平時就是一個比較樂觀的人，但我還是會盡可能地去帶給自己快樂，有時如果很忙，能好好的吃飯對我來說就是一種快樂，或是能睡滿8小時也能讓我隔天起床後心情更愉悅。

**Comment by kingsophy** [comment]
我自認是一個比較樂觀的人，但有時也會因為一些原因而陷入低潮，我的解決方式是找一個安靜的角落獨處一下。對我來說很有用，負面情緒很快就退掉了。

**Comment by leotsai940914** [comment]
《關燈會讓我快樂？！》
找到快樂的方法對我來說很簡單，我喜歡叫它充電。我這個人是很吃重氛圍的人，只要氛圍對了，做什麼都順利。
當我覺得氛圍開始往負面的方向走，例如開始自我懷疑，覺得諸事不順，或者整天耗在沒有意義的事情上的時候，這就是氛圍不對了。
這時候我會「關燈」，沒錯就是「關燈」，做什麼事情都不要開燈，例如開個小燈吃晚餐，關起燈在月光照進的房間裡聽舒緩的音樂，效果最好的是關燈洗澡。
關燈對我來說就是把眼睛這個最耗能的感官關起來，放大其他感官的感受，那是一種全新的體驗，新奇又放鬆，你會發現你感受的世界變得和眼睛看到的不同。各位也可以試試關燈，大推。

**Comment by CtyceT** [comment]
我平時在外對其他人都是用樂觀的態度面對他們，但是對於自己的事卻相當悲觀。我很願意和朋友們分享快樂，但是自己遇到低潮期的時候反而會關閉社交，讓自己獨自沈澱。記憶中快樂的事情其實很少，但傷心的事情常常就會突然閃過又被刺痛一下... 各位真的要好好珍惜身邊的人事物，畢竟生命中的無常是不會預先為我們提醒的。
